client/src/components/ElementEditor/AddNewButton.js

```diff
@@ -77,7 +77,12 @@ class AddNewButton extends Component {
   render() {
     return (
       <InputGroup className="elemental-editor__add-new-block-control">
-        <Input type="select" className="no-change-track" onChange={this.handleTypeChange}>
+        <Input
+          type="select"
+          id="elemental-editor_add-new-block-control_select-dropdown"
```

Comment:
Note that pages can have multiple elemental areas. While we haven't been strongly keeping this in mind while doing our development, this wouldn't work with two or more editors since IDs must be unique. Can you add classes instead please?